packages/@uppy/provider-views/src/ProviderView/ProviderView.tsx

```diff
@@ -333,11 +337,18 @@ export default class ProviderView<M extends Meta, B extends Body> {
     this.setLoading(false)
   }
 
-  #searchDebounced = debounce(this.#search, 500)
+  #searchDebounced = debounce(
+    this.#search,
+    (ProviderView as unknown as Record<symbol, number | undefined>)[
+      Symbol.for('uppy test: searchDebounceMs')
+    ] ?? 500,
```

Comment:
shouldn't we use `leading: true` when running tests with searchDebounceMs=0? that way it won't be delayed until the next tick, see doc: https://lodash.com/docs/4.17.15#debounce

I'm afraid that delaying until next tick could cause flakes.